manifest/com.nalathethird/IDontNeedHelp/info.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "IDontNeedHelp",
+	"id": "com.nalathethird.IDontNeedHelp",
+	"description": "A mod to remove the 'Help' Section on the Dashboard for 'Experienced Users'- only.",
+	"category": "Dashboard",
+	"sourceLocation": "https://github.com/nalathethird/Fck-HelpSection",
+	"versions": {
+		"1.0.0": {
+			"releaseUrl": "https://github.com/nalathethird/Fck-HelpSection/releases/tag/1.0.0",
+			"artifacts": [
+				{
+					"url": "https://github.com/nalathethird/Fck-HelpSection/releases/download/1.0.0/IDontNeedHelp.dll",
+					"filename": "IDontNeedHelp.dll",
+					"installLocation": "/",
```

Comment:
Same as other mod, not needed for mods in general as the correct location is the default